public/pages/SnapshotPolicies/containers/SnapshotPolicies/SnapshotPolicies.tsx

```diff
@@ -0,0 +1,454 @@
+        disabled={selectedItems.length != 1}
+        data-test-subj="editButton"
+        onClick={() => {
+          this.closePopover();
+          this.onClickEdit();
+        }}
+      >
+        Edit
+      </EuiContextMenuItem>,
+      <EuiContextMenuItem
+        key="Delete"
+        icon="empty"
+        disabled={!selectedItems.length}
+        data-test-subj="deleteButton"
+        onClick={() => {
+          this.closePopover();
+          this.showDeleteModal();
+        }}
+      >
+        <EuiTextColor color="danger">Delete</EuiTextColor>
+      </EuiContextMenuItem>,
+    ];
+    const actionsButton = (
+      <EuiButton
+        iconType="arrowDown"
+        iconSide="right"
+        disabled={!selectedItems.length}
+        onClick={this.onActionButtonClick}
+        data-test-subj="actionButton"
+      >
+        Actions
+      </EuiButton>
+    );
+    const actions = [
+      <EuiButton iconType="refresh" onClick={this.getPolicies} data-test-subj="refreshButton">
+        Refresh
+      </EuiButton>,
+      <EuiButton disabled={!selectedItems.length} onClick={this.onClickStop}>
+        Disable
+      </EuiButton>,
+      <EuiButton disabled={!selectedItems.length} onClick={this.onClickStart}>
+        Enable
+      </EuiButton>,
+      <EuiPopover
+        id="action"
+        button={actionsButton}
+        isOpen={isPopoverOpen}
+        closePopover={this.closePopover}
+        panelPaddingSize="none"
+        anchorPosition="downLeft"
+        data-test-subj="actionPopover"
+      >
+        <EuiContextMenuPanel items={popoverActionItems} />
+      </EuiPopover>,
+      <EuiButton onClick={this.onClickCreate} fill={true}>
+        Create policy
+      </EuiButton>,
+    ];
+
+    const subTitleText = (
+      <EuiText color="subdued" size="s" style={{ padding: "5px 0px" }}>
+        <p style={{ fontWeight: 200 }}>
+          Define an automated snapshot schedule and retention period with a snapshot policy.{" "}
+          <EuiLink href={SNAPSHOT_MANAGEMENT_DOCUMENTATION_URL} target="_blank">
+            Learn more
+          </EuiLink>
+        </p>
+      </EuiText>
+    );
+
+    const promptMessage = (
+      <EuiEmptyPrompt
+        style={{ maxWidth: "45em" }}
+        body={
+          <EuiText>
+            <p>{getMessagePrompt(loadingPolicies)}</p>
+          </EuiText>
+        }
+        actions={
+          <EuiButton onClick={this.onClickCreate} fill={true}>
+            Create policy
+          </EuiButton>
+        }
+      />
+    );
+
+    return (
+      <>
+        <ContentPanel title="Snapshot policies" actions={actions} subTitleText={subTitleText}>
+          <EuiSearchBar
+            box={{
+              placeholder: "e.g. ",
+              incremental: false,
+            }}
+            onChange={this.onSearchChange}
+          />
+          <EuiBasicTable
+            items={policies}
+            itemId="name"
+            columns={this.columns}
+            pagination={pagination}
```

Comment:
Good catch for this bug! 

It's because in onTableChange, from is calculated as `from: index ? (size ? index * size : prevFrom) : prevFrom,` However, index would be 0, if you select back the first page, and index ? evaluate to false and get value from `prevFrom`, so we see the page cannot be change back.